doc/mk_sdcard_image.sh

```diff
@@ -465,18 +433,51 @@ if [ -z "$doma_name" ]; then
 	exit 2
 fi
 
+# Define partitions for different products.
```

Comment:
Can we move this piece to a dedicated function on top, so it has its own space not being part of other func?
This will allow us using it w/o any further move and improve readability, e.g. once you open the script you see the image partitioning